dnf5/context.cpp

```diff
@@ -209,18 +209,41 @@ void Context::print_info(const char * msg) {
 }
 
 
-void Context::update_repo_load_flags_from_specs(const std::vector<std::string> & pkg_specs) {
+void Context::add_repo_metadata(const std::string & type) {
+    libdnf::repo::RepoQuery repos(base);
+    repos.filter_enabled(true);
```

Comment:
I would prefer to modify also disabled repository. If some plugin modify the enabled set after adding load flag it can end up with inconsistency. Proposing to remove line ` repos.filter_enabled(true);`